.github/workflows/cmake.yml

```diff
@@ -203,6 +203,16 @@ jobs:
       # This is so that device binaries can find the sycl runtime library
       run: cmake --build ${{github.workspace}}/build -j $(nproc)
 
+    - name: Information about platform
+      if: always()
+      run: |
+        export PATH=${workspace}/dpcpp_compiler/bin:$PATH
+        export CPATH=${workspace}/dpcpp_compiler/include:$CPATH
+        sycl-ls
+        lsb_release -a
+        lspci | grep VGA
+        clinfo | less
+
```

Comment:
I just realized we had a similar script in PMDK - you can gather even more info, like info about all installed packages in the OS. Also, you can make it a script in .github/scripts

 https://github.com/pmem/pmdk/blob/master/utils/gha-runners/get-system-info.sh

Reply:
I think packages are ok, you can trim env down, to only print useful variables, like `(C)PATH`,  `(LD_)LIBRARY_PATH`, etc.

Reply:
Okey